raiden/tests/conftest.py

```diff
@@ -477,7 +479,7 @@ def pytest_runtest_teardown(item):
 
     def report():
         gevent.util.print_run_info()
-        pytest.fail(
+        raise RetryTestError(
```

Comment:
IMO this should be done on timeouts of transactions, not in the report of failure.